test/OrchardCore.Tests.UI/Pages/OrchardCoreFeaturesPage.cs

```diff
@@ -0,0 +1,66 @@
+using Atata;
+using Atata.Bootstrap;
+using Lombiq.Tests.UI.Components;
+
+namespace OrchardCore.Tests.UI.Pages
+{
+    // Atata convention.
+#pragma warning disable IDE0065 // Misplaced using directive
+    using _ = OrchardCoreFeaturesPage;
+#pragma warning restore IDE0065 // Misplaced using directive
+
+    [Url("Admin/Features")]
+    public sealed class OrchardCoreFeaturesPage : OrchardCoreAdminPage<_>
```

Comment:
Is it required to create custom mapping classes? Will other more specific tests be able to use some inline html path to check for elements?